src/alembic/env.py

```diff
@@ -42,6 +43,7 @@ def run_migrations_offline() -> None:
     # url = config.get_main_option("sqlalchemy.url")
     context.configure(
         url=database.SQLALCHEMY_DATABASE_URL,
+        # url=database.SQLALCHEMY_TEST_DATABASE_URL,
```

Comment:
ah! This is a good point; we'll want to add support for testing migrations using the local test database.

Though actually, the only reason for the test database is for running things like unit & integration tests, which can freely assume the most up to data DB, so for the time being feel free to just use the main database for tests!

Reply:
WAIT, no we won't want to add support for migrations to the test database!

I totally forgot why I created a test database in the first place, so I've added it to our FAQ here https://github.com/CSSS/csss-site-backend/wiki/FAQ#6-why-is-there-a-main-database--a-test-database

The reason is that the test database is for use without migrations, so we can do testing more quickly (it ignores alembic & just wipes & recreates the database whenever `./load_test_db.py` is run)

The main database is for use with migrations.